assemblies/assembly-about-rhdh.adoc

```diff
@@ -8,15 +8,43 @@ endif::[]
 ifdef::context[]
 [id="about-rhdh_{context}"]
 endif::[]
-= About {product}
+= About {product} ({product-very-short})
 
 :context: about-rhdh
 
-{product} is a fully supported, open developer platform that reduces friction and frustration of developers while boosting productivity.
+{product} is all about making your development process smoother and more efficient. It unifies all the tools and resources your team needs into one platform, helping you collaborate better and get your applications from code to deployment faster.
 
-This platform is driven by a centralized software catalog, providing efficiency to your microservices and infrastructure.
+== Why use {product}?
 
-Use {product} to simplify decision-making through a selection of internally approved tools, programming languages, and developer resources within a self-managed portal.
+* *Boost developer productivity:* Say goodbye to common organizational hiccups. With clear guidelines and automated processes your team can focus on what they do best - writing great code.
 
+* *Unified dashboard:* Access everything in one place - from Git repositories and CI/CD pipelines to issue tracking, monitoring, and documentation. With over 150 plugins, it’s all curated to fit your company’s best practices.
 
-include::modules/discover/con-benefits-of-rhdh.adoc[leveloffset=+1]
+* *Built in best practices:* Use software templates that encode your organization’s standards, making tasks like creating new applications or setting up CI/CD pipelines a breeze.
+
+* *Easy onboarding:* New developers can get up to speed quickly, thanks to standardized tools and processes that reduce the learning curve.
+
+* *Robust security and access control:* Manage rules and permissions effectively with enterprise-grade role based access control (RBAC), keeping your projects secure.
+
+* *Scalable documentation:* Keep your code and documentation together in the same repository, so everything stays in sync without relying on separate systems.
+
+include::modules/about/con_elevate-your-development-with-secure-templates.adoc[leveloffset=+1]
+
+== Supported Platforms
+R{product} runs on {ocp-short} {ocp-version-min}-{ocp-version} and supports AMD64 and Intel 64 (x86_64) architectures. Additionally, it’s compatible with the following hosted environments:
+
+* {eks-brand-name}
+* {aks-brand-name}
+* {osd-short} on {gcp-brand-name}
```

Comment:
Is this the SAME content copied from another section (release notes?) or have you MOVED that content? Asking because if it's copied then it ought to be a snippet file that is reused rather than hardcoded into this assembly.

Reply:
@nickboldt Yes, I copied the content from release notes. Going forward, I believe the release notes will not include the About RHDH content anymore. The PR link that you provided has not been merged. Do you suggest creating a snippet file?

Reply:
@gtrivedi88 I would suggest you add [this section ](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/release_notes/index#supported-platforms) as review comments on [Judith's PR](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/661/files#diff-a0b73123541403cc024ce5b9a45f44f437770873e0ed18ad48b8bd99457157a0R7). She has already created a concept module. Let's reuse instead of duplicate content that we will need to maintain in two places.